### DISCUSSION TITLE
Dependabot creating repository labels

### DISCUSSION THREAD

**Discussion by darakeon** [community/community#43220] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

Hi. My name is Lucas. I just noticed the PRs created by dependabot had labels, but they labels that I had not created in my repository. Is there a way to disable dependabot from creating its own desired labels in my repos? Or the only way is to delete them when it creates them?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @darakeon 

You can put `labels:` and specify the names you want

![image](https://user-images.githubusercontent.com/47532703/210409131-e73ec496-6e73-4fb1-8092-e005456e12db.png)

**Comment by darakeon** [comment]
I'm Brazilian, it is a funny thing that I asked in english, you answered in english, but the comments are in portuguese...
